contracts/ownership/Ownable.sol

```diff
@@ -22,14 +22,21 @@ contract Ownable {
    * account.
    */
   constructor() public {
-    owner = msg.sender;
+    owner_ = msg.sender;
+  }
+
+  /**
+   * @return the address of the owner.
+   */
+  function getOwner() public view returns(address) {
```

Comment:
I'm not fond of starting this convention. I feel like it goes against the rest of the ecosystem.

Reply:
This is what we discussed about having verbs on function names. I won't like it, but I can remove the `get`s and try to convince the ecosystem to use them before 3.0.

Reply:
Yes, I'd prefer to remove the `get`.